### PR TITLE
Ht2/skopinov

### PR DESCRIPTION
**Pull request by Dyskop**

### CODE REVIEW

File: app/src/test/.gitignore [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # TBD
 
 # Ignore Gradle project-specific cache directory
-.gradle
+../../../.gradle
```
Comment (AlexOreshkevich) [inline app/src/test/.gitignore:5]: Why did you changed that?

File: app/src/test/java/by/academy/jc/skopinov/ht2/CalculatorTest.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+  Calculator calculator = new Calculator();
+
+  @Test
+  void sum() {
+    assertEquals(23, calculator.sum(3, 5), "the calculator is not working properly");
```
Comment (AlexOreshkevich) [inline app/src/test/java/by/academy/jc/skopinov/ht2/CalculatorTest.java:12]: Assertions should be placed in separate `@Test` methods.

**Comment by AlexOreshkevich** [comment]
graded

File: app/src/test/.gitignore (hunk 1 above)
Comment (Dyskop) [reply app/src/test/.gitignore:5]: Нas it changed? Maybe I didn't change anything there, but just put an unnecessary checkbox when I made a commit.

**Pull request closed without merging**